### ISSUE TITLE
Topic Page: Access individual Topic - Properties Tab - Edit Mode

### ISSUE THREAD

**Issue by christiemolloy** [uxd-mk-prototype/uxd-mk-data-plane-ui#8]
- #7 should be completed first before this is started.
- All fields from #7 should be editable to match screenshot.
- Jenn, since we dont have backend integration, these fields will update but that new data wont remain there after a refresh, is that ok?

**Comment by jgiardino** [comment]
> since we dont have backend integration, these fields will update but that new data wont remain there after a refresh, is that ok?

So if the user navigates to this page, makes changes, and hits the Save button, the data they entered will not be persisted on the Read view of the Properties tab? I think this is fine. It might be helpful to include a tooltip on the Save button to inform the test participant that, for this prototype, their updates aren't actually saved.

**Comment by jgiardino** [comment]
FYI, I included a separate issue to capture how we want to handle communication controls that aren't fully functional: #12

**Comment by christiemolloy** [comment]
> > since we dont have backend integration, these fields will update but that new data wont remain there after a refresh, is that ok?
> 
> So if the user navigates to this page, makes changes, and hits the Save button, the data they entered will not be persisted on the Read view of the Properties tab? I think this is fine. It might be helpful to include a tooltip on the Save button to inform the test participant that, for this prototype, their updates aren't actually saved.

When they hit the save button we can pass the data to "read-only" mode so their new details will show on that page. But if they do a hard refresh of the page the info will disappear. I dont think it should be a problem though

**Comment by jgiardino** [comment]
👍 That works. I'm also fine with making that a separate issue, depending on the effort involved in maintaining app state.

And I agree that losing data on a refresh is not a problem.
